test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -30,8 +31,6 @@ using namespace solidity::util;
 using namespace solidity::frontend::test;
 using namespace std;
 
-using Token = soltest::Token;
```

Comment:
I remove this here and used directly `soltest::Token`, because `libsolidity/ast/TypeProvider.h` seem to define `Token` that prevents the usage of `Token::<something>` in this file.